motionplan/tpspace/ptgC.go

```diff
@@ -2,23 +2,21 @@ package tpspace
 
 import (
 	"math"
-
-	rutils "go.viam.com/rdk/utils"
 )
 
 // ptgDiffDriveC defines a PTG family composed of circular trajectories with an alpha-dependent radius.
 type ptgDiffDriveC struct {
-	maxMps float64 // meters per second velocity to target
-	maxDps float64 // degrees per second of rotation when driving at maxMps and turning at max turning radius
-	k      float64 // k = +1 for forwards, -1 for backwards
+	maxMmps  float64 // millimeters per second velocity to target
+	maxRadps float64 // radians per second of rotation when driving at maxMmps and turning at max turning radius
```

Comment:
This is a pain but all the `ptg*.go` files should be consistent with `maxMMPS` `maxRPS`.  Also, are they "maxes" or just desired speeds to maintain?